app/renderer/components/navigation/navigator.js

```diff
@@ -149,6 +151,158 @@ class Navigator extends React.Component {
     ipc.off(messages.SHORTCUT_ACTIVE_FRAME_FORWARD, this.onForward)
   }
 
+  // BEM Level: navbarCaptionButtonContainer__navbarMenubarFlexContainer__navigatorWrapper
+  get navigatorWrapper () {
+    return <div className={cx({
+      navigatorWrapper: true,
+      [css(styles.navigatorWrapper)]: true
+    })}
+      onDoubleClick={this.onDoubleClick}
+      onDragOver={this.onDragOver}
+      onDrop={this.onDrop}
+    >
+      {this.topLevelStartButtons}
+      <NavigationBar />
+      {this.topLevelEndButtons}
+    </div>
+  }
+
+  // BEM Level: navbarCaptionButtonContainer__navbarMenubarFlexContainer__navigatorWrapper__topLevelStartButtons
```

Comment:
just an idea despite being not sure how to implement this: require a BEM style map file to be implemented, grammar should be linted, and check if the BEM structure is maintained.

````
topLevelStartButtons: {
    parent: 'navbarCaptionButtonContainer__navbarMenubarFlexContainer__navigatorWrapper__topLevelStartButtons',
    files: 'navigator.js',
    ....
},
````

````
const navigationBemMap = require('/style/stylemaps/navigation.map')
````